cmd/helper/authHelp.go

```diff
@@ -28,22 +29,34 @@ import (
 
 	"github.com/spf13/cobra"
 	"github.com/zalando/go-keyring"
-	"github.com/zmb3/spotify"
+	"github.com/zmb3/spotify/v2"
+	"github.com/zmb3/spotify/v2/auth"
 	"golang.org/x/oauth2"
 )
 
 const redirectURI = "http://localhost:8888/callback"
 
 var (
-	authenticator = spotify.NewAuthenticator(redirectURI, spotify.ScopeStreaming, spotify.ScopeUserModifyPlaybackState, spotify.ScopeUserReadPlaybackState, spotify.ScopePlaylistModifyPrivate, spotify.ScopePlaylistModifyPublic)
-	ch            = make(chan *spotify.Client)
-	clientID      string
-	secret        string
-	state         = "ringdingthing"
+	authenticator = spotifyauth.New(
+		spotifyauth.WithRedirectURL(redirectURI),
+		spotifyauth.WithClientID(clientID),
```

Comment:
When the `spc auth` command generates a URL, it doesn't generate it with a clientID. I think this is because, when this `spotifyauth.New()` function is called, `clientID` isn't yet defined. I'm not sure what the `spotifyauth.New()` function is doing behind the scenes, but maybe using a pointer to a string could do it?